plugin/symbols.py

```diff
@@ -156,3 +159,61 @@ def process_symbol_informations(self, items: List[Dict[str, Any]]) -> List[List[
             self.regions.append((range_to_region(Range.from_lsp(item['location']['range']), self.view), None))
             quick_panel_items.append([item['name'], format_symbol_kind(item['kind'])])
         return quick_panel_items
+
+
+class SymbolQueryInput(sublime_plugin.TextInputHandler):
+
+    def validate(self, txt: str) -> bool:
+        return txt != ""
+
+    def placeholder(self) -> str:
+        return "Symbol"
+
+
+class LspWorkspaceSymbolsCommand(LspTextCommand):
+    def __init__(self, view: sublime.View) -> None:
+        super().__init__(view)
```

Comment:
Should skip overriding if you are not doing anything extra.
Or is it some style choice?

Reply:
Done